javascript-forms/form-values.js

```diff
@@ -0,0 +1,15 @@
+function getData(event) {
+  console.log('submit');
+
+  event.preventDefault();
+  var one = xx.elements.name.value;
+  var two = xx.elements.email.value;
+  var three = xx.elements.message.value;
+  var object = { name: one, email: two, message: three };
```

Comment:
these could use better names too. Get in the habit of giving your variables and functions quality names.